api_version.go

```diff
@@ -7,5 +7,5 @@
 package stripe
 
 const (
-	apiVersion string = "2024-06-20"
+	apiVersion string = "2024-09-30.acacia"
```

Comment:
I changed this to get my local example working, but am not sure it should be included in this PR

Reply:
change the base of the PR from master to sdk-release/next-major - that branch has this change

Reply:
good call! Changed the base branch, so this will be a no-op. I'm not sure GH will reflect that though